### PR TITLE
[lip-6] Edits

### PR DESCRIPTION
**Pull request by DavidLDill**
I've mostly tried to do some wordsmithing.

There are some "TODO"s embedded in paragraphs [TODO... ] for points that I didn't know how to resolve.

The properties are still drafty until I make more progress in formal
specification and verification.

### CODE REVIEW

File: lips/lip-6.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 lip: 6
 title: Libra Validator Configuration Management
-authors: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
+author: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
```
Comment (JoelMarcey) [inline lips/lip-6.md:4]: This has to stay `authors` when there are multiple authors as that helps with the parsing via the `,` separator.

**Comment by JoelMarcey** [comment]
@DavidLDill `bors-libra` is kinda broken for this repo due to how we are doing branch protections and requirements, I believe. So we can just directly merge it for now, if you want.

cc @bmwill

**Comment by bmwill** [comment]
If we remove the requirement on deploy-preivew check and I do a reconfig it should work properly. Just let me know if you want me to do that

**Comment by JoelMarcey** [comment]
@bmwill I want to come up with a way to remove the check for `bors` but have the check so that no one accidentally lands a broken site. I want my cake and I want to eat it too.

**Comment by DavidLDill** [comment]
I don't know what to do to make this land, and Lera would like to work on it.  So, any help you can give to get it landed would be great.

**Comment by JoelMarcey** [comment]
@DavidLDill merged!

**Pull request merged**